course_catalog/serializers.py

```diff
@@ -776,6 +781,13 @@ def update(self, instance, validated_data):
                 stafflist = super().update(instance, validated_data)
                 if topics_data is not None:
                     stafflist.topics.set(CourseTopic.objects.filter(id__in=topics_data))
+                if (
+                    instance.items.exists()
+                    and instance.privacy_level == PrivacyLevel.public.value
+                ):
+                    upsert_staff_list(stafflist.id)
+                else:
+                    delete_staff_list(stafflist)
```

Comment:
This initially confused me...I did not pay enough attention to the imports, so my I thought "Why are we deleting stafflists on update if they are not public?". Then from imports I saw this isn't deleting the stafflist record, just removing it from search.

Do you think it's worth importing the `search.task_helpers` so the usage is `search.task_helpers. delete_staff_list()` and more clearly search related? Or some other solution.

Anyway...not a big deal!t...the IDE hover-docstrings alleviated my confusion pretty quickly. But if you think of some easy resolution, 👍

Reply:
Good idea, I'll make a new issue to rename the `search.task_helpers` functions to be more obvious in general about what they do (like `delete_userlist` -> `userlist_index_delete` and `upsert_userlist` -> `userlist_index_upsert`)